<launch>

<node name="pick_place" pkg="ur5_gripper_control" type="pick_place.py" respawn="false"/>
<node name="publish_octomap_cloud" pkg="ur5_gripper_control" type="publish_octomap_cloud.py" respawn="true"/>
    
</launch>